src/ontology/uberon-edit.obo

```diff
@@ -206,7 +205,6 @@ treat-xrefs-as-reverse-genus-differentia: HAO part_of NCBITaxon:7399
 treat-xrefs-as-reverse-genus-differentia: HBA part_of NCBITaxon:9606
 treat-xrefs-as-reverse-genus-differentia: HsapDv part_of NCBITaxon:9606
 treat-xrefs-as-reverse-genus-differentia: MA part_of NCBITaxon:10090
-treat-xrefs-as-reverse-genus-differentia: MBA part_of NCBITaxon:10090
```

Comment:
I understand you think you know what this does, but I just want to note I dont.

Reply:
This instructs the `make-bridge-ontologies-from-xrefs.pl` script to extract cross-references to MBA-prefixed terms and to generate Uberon/CL bridges to MBA (`uberon-bridge-to-mba.obo`, `cl-bridge-to-mba.obo`, etc.). Removing the line disables the generation of such bridges, which is what is wanted here since we are replacing them with custom, externally maintained bridges.